### DISCUSSION TITLE
What happened to "Move tabs that matched URLs to this panel" in v5?

### DISCUSSION THREAD

**Discussion by EntropyWorks** [mbnuqw/sidebery#910] [Q&A] [answered]
I recent wanted to try Sidebery v5 but notice I can no longer auto move matched URLs to panels. Has this feature been dropped completely?

I have multiple containers, mostly to deal with login into different cloud providers, AWS, Azure, and GCP but maybe not in the way you think. I have nearly 50 cloud accounts in AWS alone. Each login get a container. I was matching the URL and placing all the AWS different logins into one Sidebery panel.  That no longer works in v5.

**Comment by mbnuqw** [comment, marked as answer]
Well this feature was, indeed, a candidate for removal, but right now I'm planning to add a more general feature in sidebery as a replacement for it. It will be a global tab rules list, and you will be able to set a match rule (e.g. regex for URL) and an action rule for matched tab - reopen in some container, move to some panel, move to some tabs branch. All of that with more user-friendly way to adding/removing rules.

**Reply by EntropyWorks** [reply]
Thanks for the update. I guess I'm going to switch back to v4 for the moment.  It seems most people want to just container one or two things. I know I'm an edge case. I have tried other tab management extensions and this one has the kind of grouping I was hunting for.  Being able to place sites based on the URL into Groups / Panels.

My current active extension:

- [Container Tabs Sidebar](https://addons.mozilla.org//firefox/addon/containertabssidebar%40maciekmm.net)
- [KeePass Helper - URL in title](https://addons.mozilla.org//firefox/addon/%7Be56fa932-ad2c-4cfa-b0d7-a35db1d9b0f6%7D)
- [HTTPS Everywhere](https://addons.mozilla.org//firefox/addon/https-everywhere%40eff.org)
- [aws-saml-search](https://addons.mozilla.org//firefox/addon/%7B2527b456-c172-48a9-9b06-a8c26790d9b8%7D)
- [Pay by Privacy.com](https://addons.mozilla.org//firefox/addon/privacy%40privacy.com)
- [Temporary Containers](https://addons.mozilla.org//firefox/addon/%7Bc607c8df-14a7-4f28-894f-29e8722976af%7D)
- [Open in Browser](https://addons.mozilla.org//firefox/addon/openinbrowser%40www.spasche.net)
- [RESTClient](https://addons.mozilla.org//firefox/addon/%7Bad0d925d-88f8-47f1-85ea-8463569e756e%7D)
- [IBM Aspera Connect](https://addons.mozilla.org//firefox/addon/connect%40aspera.ibm.com)
- [KeeWeb Connect](https://addons.mozilla.org//firefox/addon/keeweb-connect%40keeweb.info)
- [Link Cleaner](https://addons.mozilla.org//firefox/addon/%7B6d85dea2-0fb4-4de3-9f8c-264bce9a2296%7D)
- [MarkDownload - Markdown Web Clipper](https://addons.mozilla.org//firefox/addon/%7B1c5e4c6f-5530-49a3-b216-31ce7d744db0%7D)
- [Tab Stash](https://addons.mozilla.org//firefox/addon/tab-stash%40condordes.net)
- [Firefox Multi-Account Containers](https://addons.mozilla.org//firefox/addon/%40testpilot-containers)
- [uBlock Origin](https://addons.mozilla.org//firefox/addon/uBlock0%40raymondhill.net)
- [Sidebery](https://addons.mozilla.org//firefox/addon/%7B3c078156-979c-498b-8990-85f7987dd929%7D)

```shell
# one way to get a list of active extensions
cd ~/.mozilla/firefox/ActiveProfile.default-esr
jq --arg lang "$lang" -j '.addons |
   map(select(.type == "extension" and .location == "app-profile" and (.userDisabled | not)))
   | .[] | "- [",.defaultLocale.name,"]",
     "(https://addons.mozilla.org/\($lang)/firefox/addon/",(.id|@uri),")\n" ' \
  extensions.json
```

BTW: When removing v5 and switching back to v4 on a Mac, Firefox Developer Edition, it looks like the CSS styling is lost. Have a quick hint to what files I need to wipe so v4 displays properly again?

**Reply by mbnuqw** [reply]
You can open styles editor and uncomment your custom CSS (on upgrading to v5 sidebery comment out custom CSS to avoid styles breakage). Or use your backup saved before upgrading to beta version.
